OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -194,13 +194,13 @@ private void Pane_DblClickHandler(object sender, EventArgs e)
         {
             try
             {
-                ExpandedImageForm expandedImageForm;
+                ExpandedImageDialog expandedImageForm;
                 PictureBox pictureBox = sender as PictureBox;
                 Bitmap image = (Bitmap)pictureBox.Image;
 
                 Mat mat = OpenCvSharp.Extensions.BitmapConverter.ToMat(image);
 
-                expandedImageForm = new ExpandedImageForm(mat);
+                expandedImageForm = new ExpandedImageDialog(_meterPane1.SourceImage, mat);
```

Comment:
I don't think having the source image will be necessary but having the base image you're beginning to manipulate will be. I'm also not keen on hard coding something into this event handler after I took the time to make it generic.